src/client.rs

```diff
@@ -0,0 +1,2 @@
+pub mod config;
```

Comment:
Starting the move toward pulling things out of `http` and into `server` and `client`